app/src/main/java/com/ozsaat/feeddit/IconNavigationItem.java

```diff
@@ -0,0 +1,29 @@
+package com.ozsaat.feeddit;
+
+
+import android.graphics.drawable.Drawable;
+
+public class IconNavigationItem extends NavigationItem {
+    public IconNavigationItem(String title) {
```

Comment:
This constructor is not necessary, because you will always need to have iconResId for this class